app/models/notification.rb

```diff
@@ -39,4 +39,12 @@ def kept?
   def for_feed
     attributes.slice(*FEED_ATTRIBUTES)
   end
+
+  def mailer
+    if topic.is_a?(PersonEscortRecord)
```

Comment:
do we also need to consider YouthRiskAssessment? we send a `confirm_youth_risk_assessment` webhook and `confirm_youth_risk_assessment` email

Reply:
It's only possible to create a notification with a topic of PER or Move - for everything else the topic ends up being delegated to the relevant move.

Reply:
👍